File: 1159/c/1159.c
#include <stdio.h>

int main(){
  int n, i;
  int check = 0;
  char name[30];
  int count[26]={0};

  scanf("%d", &n);

  for(i = 0; i < n; i++){
    scanf("%s", name);
    count[name[0] - 97]++;
  }

  for(i = 0; i < 26; i++){
    if(count[i] >= 5){
      check++;
      printf("%c", i + 'a');
    }
  }

  if(check == 0){
    printf("PREDAJA");
  }
  return 0;
}

File: 1159/python/1159.py
from collections import Counter

check = 0
name = []
count =[]
result = []

n = int(input())

for i in range(n):
  name = input()
  count.append(name[0])

  name_count = Counter(count)
for i in name_count:
  if name_count[i] >= 5:
    result.append(i)

if len(result) > 0:
  print(''.join(sorted(result)))
else:
  print("PREDAJA")